components/image-releaser/app.yaml

```diff
@@ -1,24 +0,0 @@
-apiVersion: 0.1.0
```

Comment:
Image releaser has been migrated to tekton? Or kubeflow won't release images officially?

Reply:
Just notice this is mentioned in the description.